### PR TITLE
section

### PR DESCRIPTION
**Pull request by Muhammadali2oo6**

### CODE REVIEW

File: src/components/section/Section.jsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                <div className="button">
+                  <button>Order Now</button>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    </div>
```
Comment (samandar0806) [inline src/components/section/Section.jsx:93]: Prietterni togirlab qoy!!!!!!!!!!!!!!!!!!!!!!!!

File: src/components/section/section.css [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+}
+body {
+  /* background-image: url(../img/game-backgrounds.webp); */
+  background-color: rgb(69, 91, 84);
+}
+.media {
```
Comment (samandar0806) [inline src/components/section/section.css:13]: nima bu media davomi bormi mediani!!!!

File: src/components/section/section.css [hunk 3]
```diff
@@ -0,0 +1,155 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
```
Comment (ibrohim5009) [inline src/components/section/section.css:1]: nimaga css ????
image

**Review by samandar0806 — CHANGES_REQUESTED** [review]

File: src/components/section/section.css (hunk 3 above)
Comment (ibrohim5009) [reply src/components/section/section.css:1]: cssdayam shunchalikmi

**Review by ibrohim5009 — CHANGES_REQUESTED** [review]

File: src/components/section/section.css [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+.img2 {
+  display: flex;
+  margin-top: 10px;
+}
+.img2 img {
+  height: 310px;
+  border-radius: 8px;
+}
+.img3 {
+  margin-right: 22px;
```
Comment (samandar0806) [inline src/components/section/section.css:45]: shu joyiga marginni o'zini ishlat

**Review by samandar0806 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**